datanode/gateway/graphql/schema.graphql

```diff
@@ -545,23 +555,23 @@ type Query {
     pagination: Pagination
   ): MarketDataConnection
 
-  "Get historical balances for an account within the given date range"
-  balanceChanges(
```

Comment:
Sorry might be missing some context here, but why is the `balanceChanges` query being removed? This is a breaking change for the frontend

Reply:
@mattrussell36 I believe it's just moved rather than removed. Look up to line 409

Reply:
Ah my mistake

Reply:
They haven't been removed, I had to re-order it because everything else had been changed so the API's were listed in alphabetical order and this one was in the middle of the pack and out of sequence.